### PR TITLE
updated tickercards

### PR DESCRIPTION
**Pull request by beckskelly**
Had  go updating the tickercards,-
1. made the logos a bit bigger
2. put %change first and in bold, some logos are still a bit problematic, like the ones with a big white space behind them (looking at you, apple!)

image

### CODE REVIEW

File: src/components/stockDiscoveryComponents/tickercard/Tickercard.js [hunk 1]
```diff
@@ -26,19 +26,39 @@ function TickerCard({ stock }) {
...
+                                                    display: "block",
+                                                    objectFit: "contain",
+                                                    paddingTop: "5px",
+                                                    marginLeft: "auto",
+                                                    marginRight:"auto"
+
+                                                }} alt="company logo"></img>
+                                            </div>
+                                        </div>
+        {/* <Container className="tickerCardImgContainer" style={{ margin: "none", paddingLeft: 0, paddingRight: 0, paddingTop: 0 }}>
```
Comment (WarrenK-design) [inline src/components/stockDiscoveryComponents/tickercard/Tickercard.js:48]: I think we can delete these if not being used anymore

**Review by WarrenK-design — CHANGES_REQUESTED** [review]
Hi Becks, 

Looks good cheers for that, just small thing getting few warnings for unused components/duplicate keys. 
image

# Formattting 
Just small thing too we noticed on Friday with numbers not being formatted correctly. 
image

Could you change this code (Lines 13 to 24). 
```
function redOrGreen() {
    if (parseFloat(stock.daily_change.percentageChange) >= 0) {
      postiveSymbol = "+"
      absoluteChange = "$" + parseFloat(stock.daily_change.absoluteChange).toFixed(2)
      return "green"
    }
    else {
      var myString = String(stock.daily_change.absoluteChange)
      absoluteChange = myString.slice(0, 1) + "$" + myString.slice(1)
      return "red"
    }
  }
```
To code below, think it will solve it. 
```
function redOrGreen() {
    absoluteChange = parseFloat(stock.daily_change.absoluteChange).toLocaleString('en-US', { style: 'currency', currency: 'USD' })
    if (parseFloat(stock.daily_change.percentageChange) >= 0) {
      postiveSymbol = "+"
      return "green"
    }
    else {
      return "red"
    }
  }
```

**Comment by beckskelly** [comment]
Cheers @WarrenK-design , good spots, those changes should be pushed now

**Review by WarrenK-design — APPROVED** [review]
Looks good,  cheers Becks

**Comment by beckskelly** [comment]
Swapped the card info around in line with the chat in discord and with other trading platforms: 

image

**Review by joeycor92 — APPROVED** [review]
Happy with this now. From our chat yesterday it does make sense to have the absolute change first! Thanks.

**Pull request merged**